dll/win32/newdev/newdev.c

```diff
@@ -366,12 +372,20 @@ SearchDriverRecursive(
         ok = FindNextFileW(hFindFile, &wfd))
     {
 
+        /* Filename is too long to be searched */
+        if (wcslen(wfd.cFileName) >= _countof(FileName))
+            continue;
+
         wcscpy(FileName, wfd.cFileName);
```

Comment:
It seems that the FileName buffer isn't needed at all in the code, and one could directly use `wfd.cFileName` for the IsDots function and the others below.
If this is indeed so, then the `wcslen() >= _countof()` test above would become useless.